web-client/src/stories/theme.stories.ts

```diff
@@ -0,0 +1,9 @@
+import { Meta } from '@storybook/angular';
+
+export default {
+  title: 'Theme',
+} as Meta;
+
+export const Theme = () => ({
```

Comment:
GitHub Actions/test is complaining about a missing return type on function

Reply:
Good catch!

[c6c26b9..6c48d84](https://github.com/registreerocks/nautilus-wallet/compare/c6c26b9d8e91a25b99c46d8287b6ab4afbcec82b..6c48d8466958e5ff17a4264d77be3d28b5615520) should resolve it: how does that look?